From 082e2a7d2738f559dbfce090c6983c5e7fbff1b1 Mon Sep 17 00:00:00 2001
From: Stone Tickle <lattis@mochiro.moe>
Date: Fri, 5 Jun 2020 12:51:25 +0900
Subject: [PATCH 2/4] set O_NONBLOCK on repeat timerfd

---
 src/wl_init.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/wl_init.c b/src/wl_init.c
index 7d9f8221..6616c703 100644
--- a/src/wl_init.c
+++ b/src/wl_init.c
@@ -1214,8 +1214,9 @@ int _glfwPlatformInit(void)
     _glfwInitTimerPOSIX();
 
     _glfw.wl.timerfd = -1;
-    if (_glfw.wl.seatVersion >= 4)
-        _glfw.wl.timerfd = timerfd_create(CLOCK_MONOTONIC, TFD_CLOEXEC);
+    if (_glfw.wl.seatVersion >= 4) {
+        _glfw.wl.timerfd = timerfd_create(CLOCK_MONOTONIC, TFD_CLOEXEC | TFD_NONBLOCK);
+    }
 
     if (_glfw.wl.pointer && _glfw.wl.shm)
     {
-- 
2.32.0

